From 739cffb4fda61669cd46038707242e5b5255ea51 Mon Sep 17 00:00:00 2001
From: Rasmus Villemoes <rv@rasmusvillemoes.dk>
Date: Wed, 12 Jun 2024 11:10:42 +0200
Subject: [PATCH] make: Explicitly specify output directory for makeinfo --html

The default output directory differs between makeinfo versions.

Equivalent to upstream commit accf5ce.

Signed-off-by: Rasmus Villemoes <rv@rasmusvillemoes.dk>
---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index 8be63fd..6272487 100644
--- a/Makefile
+++ b/Makefile
@@ -109,7 +109,7 @@ HTML_FIXUP_MENU   = '/<\/body>/i<div id="s-css-s--menu"><\/div>'
 
 html-dir: $(PKG).texi
 	@printf "Generating $(PKG)/*.html\n"
-	@$(MAKEINFO) --html $(MANUAL_HTML_ARGS) $<
+	@$(MAKEINFO) --html -o $(PKG)/ $(MANUAL_HTML_ARGS) $<
 	@for f in $$(find $(PKG) -name '*.html') ; do \
 	sed -i -e $(HTML_FIXUP_CSS) -e $(HTML_FIXUP_ONLOAD) -e $(HTML_FIXUP_MENU) $$f ; \
 	done
-- 
2.45.2

